### DISCUSSION TITLE
carto style

### DISCUSSION THREAD

**Discussion by swedneck** [enzet/map-machine#72] [Ideas]
hi, is it possible to use the carto style (and other styles in general) with roentgen?

**Comment by enzet** [comment]
Hello!

Thank you for the question!

If you mean MapCSS, the answer is unfortunately no, for now, MapCSS isn't supported.  Can you describe the use case?  You want the Röntgen renderer to have the possibility to produce the image as close as possible to carto style, or exactly the same image as in carto?

I have plans to support MapCSS parsing but I'm afraid it's not possible with Röntgen to produce exactly the same image as Mapnik for carto style.

BTW, Röentgen now supports the quite opposite feature.  Röntgen map style with custom icons can be converted into a MapCSS file and be used by Mapnik, JOSM, and other renderers.

**Comment by swedneck** [comment]
My use case is simply rendering tiles without having to suffer through the normal setup used for tile rendering (involving postgres among other things, which is way more effort than i have the energy for).

Ideally it would be identical to carto, but it's certainly not a dealbreaker if there are differences.

**Comment by enzet** [comment]
Yes, I know what are you talking about, dealing with Postgres and everything to just create tiles is overhead.  But I'm afraid Röntgen is not what you want, since creating an alternative for the Mapnik never was the goal.  Tile generation in Röntgen is quite slow (precisely because it's Python and doesn't use databases) and mostly aims to support its own map style.

But if the current state of the Röntgen style suits you and speed is not critical, you can generate SVG tiles.
